doc/nrf/known_issues.rst

```diff
@@ -1442,7 +1443,7 @@ NCSDK-16644: :ref:`caf_sensor_manager` macro incorrectly converts float to senso
 
 .. _ncsidb_925:
 
-.. rst-class:: v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0
+.. rst-class:: v2-3-0 v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0
```

Comment:
Should be fixed in NCS 2.3.
@rakons, could you double check, please ?

Reply:
Yes. Fixed by https://github.com/nrfconnect/sdk-nrf/pull/10093